tasks/lib/packages/installed_packages.js

```diff
@@ -1,7 +1,37 @@
-import { relative } from 'path';
+import { relative, resolve } from 'path';
+import { readFileSync } from 'fs';
 
 import { callLicenseChecker } from './license_checker';
 
+function resolveLicense(licenseInfo, key, licenseOverrides) {
+  const {
+    private: isPrivate,
+    licenses: detectedLicenses,
+    realPath,
+  } = licenseInfo[key];
+
+  // `license-checker` marks all packages that have `private: true`
+  // in their `package.json` as "UNLICENSED", so we try to lookup the
+  // actual license by reading the license field from their package.json
+  if (isPrivate && detectedLicenses === 'UNLICENSED') {
+    try {
+      const pkg = JSON.parse(readFileSync(resolve(realPath, 'package.json')));
+      if (!pkg.license) {
+        throw new Error('no license field');
+      }
+      return [pkg.license];
```

Comment:
In practice this lgtm but noting we get a mapped license https://github.com/davglass/license-checker/blob/master/lib/license.js.  Which will then most likely be caught by the licenses overrides check.

Reply:
Yeah, since there shouldn't be a way to get `private: true` modules from outside Kibana I'm not worried about mapping licenses or supporting other license fields.